### PR TITLE
add woocommerce

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -98,13 +98,69 @@ services:
...
+      LETSENCRYPT_EMAIL: ${LETSENCRYPT_EMAIL:-<no value>}
+      VIRTUAL_HOST: ${WOOCOMMERCE_BLOG_HOST}
+    expose:
+      - "80"
+    links:
+      - mariadb_blog
+    volumes:
+      - "woocommerce_blog_html:/var/www/html"
+
+  mariadb_shop:
```
Comment (NicolasDorier) [inline docker-compose.yml]: no need of having 2 mariadb, can just use one maria server but different DB.

Comment (Kukks) [reply docker-compose.yml]: Are they just created on demand?

Comment (NicolasDorier) [reply docker-compose.yml]: yes

**Comment by Kukks** [comment]
Ok check it out now

**Pull request merged**